### ISSUE TITLE
Locate biome range distance issue

### ISSUE THREAD

**Issue by TheLemzz** [HimmelKreis4865/JavaGen#14]
![image](https://github.com/HimmelKreis4865/JavaGen/assets/75723466/05c4ef60-f4b7-4736-b7cb-4e8faf6b136a)
why /locate rande distance so small? ~134 blocks

**Comment by HimmelKreis4865** [comment]
where is the problem? Is this wrong? because the distance just represents the air - line distance to the point...the y axis might be wrong, shouldn't make a big difference iirc

**Comment by TheLemzz** [comment]
It's not about Y, it's about the fact that locate can't find anything beyond ~200 blocks, whereas on JAVA locate finds biomes/structures up to a few thousand blocks away

**Comment by HimmelKreis4865** [comment]
yeah this doesnt make any sense cuz its using the same 100 chunk radius that is also used in the java client. im gonna look into this tomorrow

**Comment by TheLemzz** [comment]
it happend only with biomes.
